### PR TITLE
[SMALLFIX] Refactor advance and move block in stream.

### PR DESCRIPTION
**Pull request by calvinjia**
Moves the common logic into a new method `updateBlockInStream`.

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java [hunk 1]
```diff
@@ -312,4 +283,31 @@ private void moveBlockInStream(long newPos) throws IOException {
...
+  private void updateBlockInStream(long blockId) throws IOException {
+    if (mCurrentBlockInStream != null) {
+      mCurrentBlockInStream.close();
+    }
+    try {
+      mCurrentBlockInStream = mContext.getTachyonBlockStore().getInStream(blockId);
+      mShouldCacheCurrentBlock =
+          !(mCurrentBlockInStream instanceof LocalBlockInStream) && mShouldCache;
+    } catch (IOException ioe) {
+      if (null == mUfsPath || mUfsPath.isEmpty()) {
```
Comment (yupeng9) [inline clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java]: NIT: mUfsPath == null

**Comment by jsimsa** [comment]
LGTM +1

File: clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java (hunk 1 above)
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java]: Updated, thanks.

**Comment by yupeng9** [comment]
+1 LGTM

**Pull request merged**